src/navigation/MainNavigator.tsx

```diff
@@ -1,7 +1,8 @@
 import React from 'react';
 import {createStackNavigator} from '@react-navigation/stack';
 import {NavigationContainer} from '@react-navigation/native';
-import {Connections} from 'src/screens';
+import {Connections, Booking} from 'src/screens';
```

Comment:
This is the other place where it's not following the '@' format. I don't know if there's a reason for that, so I'll just add the suggestion, but just let me know if it's meant to be like that! 😄 
```suggestion
import {Connections, Booking} from '@screens';
```